BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -133,14 +162,16 @@ final class SignUpViewController: BaseViewController, SignUpViewProtocol {
             make.centerX.equalTo(safeArea)
         }
         
-        logoImageView.snp.makeConstraints { make in
-            make.size.equalTo(196)
-            make.centerX.equalTo(safeArea)
+        slidesCollectoinView.snp.makeConstraints { make in
+            make.leading.equalTo(safeArea.snp.leading)
+            make.trailing.equalTo(safeArea.snp.trailing)
             make.centerY.equalTo(backgroundImageView.snp.top)
+            make.height.equalTo(LayoutConstants.ImageLogo.height)
         }
         
-        logoLabel.snp.makeConstraints { make in
-            make.centerX.centerY.equalTo(logoImageView)
+        pageControl.snp.makeConstraints { make in
+            make.centerX.equalTo(safeArea)
+            make.bottom.equalTo(phraseLabel.snp.top).inset(-30)
```

Comment:
.offset(30)?

Reply:
.offset(30) does the same in this case. Constraint was changed to equalTo(phraseLabel.snp.bottom).inset(93)